spec/presenters/hyrax/honors_college_thesis_presenter_spec.rb

```diff
@@ -25,20 +26,14 @@
   let(:solr_properties) do
     %w[contributor_advisor contributor_committeemember degree_discipline degree_field degree_grantors degree_level degree_name graduation_year]
```

Comment:
The fix for this is to change `%w` to `%i` and switch to the commented `expect` statement

Reply:
change made and it passes! yay!